src/components/screens/SearchVariantsScreen.vue

```diff
@@ -0,0 +1,441 @@
+          <Button class="p-button-plain" @click="fuzzySearch">Search</Button>
+        </div>
+        <div class="mavedb-search-form-view-switch">
+          Click here to return to HGVS search:
+          <Button class="p-button-plain" @click="hgvsSearchVisible = true; fuzzySearchVisible = false">HGVS Search</Button>
+        </div>
+      </div>
+      <p v-if="alleles.length === 0">
+        Enter an HGVS string or variant definition above to obtain a variant and its details.
+      </p>
+      <div v-for="allele in alleles" :key="allele.clingenAlleleId" class="col-12">
+        <Card>
+          <template #content>
+            <div class="variant-search-result">
+              <div v-if="allele.variants.length > 0" class="variant-search-result-button">
+                <router-link :to="`/variants/${allele.clingenAlleleId}`">
+                  <Button label="View in MaveMD" icon="pi pi-eye" />
+                </router-link>
+              </div>
+              <div class="variant-search-result-content">
+                <!-- TODO handle no canonical allele name or just leave blank? -->
+                <div class="variant-search-result-title">
+                  {{ allele.canonicalAlleleName }}
+                </div>
+                <div v-if="allele.clingenAlleleUrl" class="variant-search-result-subcontent">
+                  ClinGen Allele ID:
+                  <a :href="allele.clingenAlleleUrl">{{ allele.clingenAlleleUrl }}</a>
+                </div>
+                <div v-if="allele.grch38Hgvs" class="variant-search-result-subcontent">
+                  GRCh38 coordinates: {{ allele.grch38Hgvs }}
+                </div>
+                <div v-if="allele.grch37Hgvs" class="variant-search-result-subcontent">
+                  GRCh37 coordinates: {{ allele.grch37Hgvs }}
+                </div>
+                <div v-if="allele.maneCoordinates.length > 0" class="variant-search-result-subcontent">
+                  MANE transcript coordinates:
+                  <DataTable striped-rows showGridlines size="small" :value="allele.maneCoordinates">
+                    <Column field="sequenceType" header="Sequence Type"></Column>
+                    <Column field="database" header="Database"></Column>
+                    <Column field="hgvs" header="Coordinates"></Column>
+                  </DataTable> 
+                </div>
+                <div v-if="allele.variantsStatus == 'Loaded' && allele.variants.length > 0" class="variant-search-result-subcontent">
+                  MaveDB score sets containing variant:
+                  <li v-for="variant in allele.variants" :key="variant.urn">
+                    <router-link :to="{name: 'scoreSet', params: {urn: variant.scoreSet.urn}, query: {variant: variant.urn}}">
+                      {{ variant.scoreSet.title }}
+                    </router-link>
+                  </li>
```

Comment:
To save some screen real estate, we might consider only optionally expanding this list.